Plugins/Configuration Migration Plugin/README.md

```diff
@@ -194,6 +194,10 @@ Below the full list of configurations in scope:
 * **Query Templates** (service: `templates`, requires `custom_groups` and the configuration of a *Service Account*): not private Query Templates (i.e. shared with everyone or specific groups); the sharing option will synchronized as well
   
   > If an existing Query Template is unshared in the source system, it will not be in the export file anymore. A successive import of will mark this change as a `delete` action and delete the item from the `destination` system.
+
+* **Query Tags** (service: `tags`, requires the configuration of a *Service Account* with *Can create read-only queries and run queries* access rights): all query tags
+  > Queries that were assigned tags before the migration, will still have the same tags assigned after the migration.
```

Comment:
Unless I'm wrong, queries now have a dependency on tags. Thus:
* I think query tags are also migrated, so this comment looks a bit confusing to me.
* This dependency should be documented above on the **Queries** section (`...requires custom_groups ++and tags++...`).

It's worth testing the actual plugin behaviour, I'm not 100% sure about these claims.

Reply:
@gmasclet 
Its mentioned that `all query tags` are migrated, perhaps we need to make it more explicit

Also tags are not a mandatory dependency for queries, Graph queries can be created, updated without query tags, unlike custom group that is a required dependency that is why i didn't document it

I already tested this behavior, we can migrate queries normally that has no tags assigned to it